prod-refractr.yml

```diff
@@ -22,6 +22,11 @@ refracts:
 - firefox-settings-attachments.cdn.allizom.org/:
   - firefox-settings-attachments-cdn.stage.mozaws.net
 
+  # SVCSE-1586
+- accounts.firefox.com/:
```

Comment:
Just to make sure you want to redirect accounts.mozilla.org and accounts.mozilla.com toaccounts.firefox.com?  I have a feeling the logic here might be inversed based on what you expect. 

If you look a little futher down there are some examples with some tests which might help clarify.

Reply:
Yes, this is to redirect accounts.mozilla.org and accounts.mozilla.com to accounts.firefox.com (see https://mozilla-hub.atlassian.net/browse/SVCSE-1586 for context)